src/lazy/expanded/macro_table.rs

```diff
@@ -490,7 +507,7 @@ impl MacroTable {
         }
     }
 
-    pub fn add_macro(&mut self, template: TemplateMacro) -> IonResult<usize> {
+    pub fn add_template_macro(&mut self, template: TemplateMacro) -> IonResult<usize> {
```

Comment:
🪧 As mentioned prior, this was renamed to accommodate the introduction of a more general `append_macro` method (see below).